etc/config.xml

```diff
@@ -16,6 +16,9 @@
                 <api_refund_url></api_refund_url>
                 <api_key></api_key>
             </general>
+            <advance>
+                <list_mode>Production</list_mode>
```

Comment:
This needs to be `prod` in order to work with existing logic.

Reply:
Hi @soleilcot , thanks for your comment. The code is correct and working properly as I've tested with other environments. All you need to do is try out this and if it's not work for you then do comment here or on an asana ticket with a video recording. It does work at my end so it should work globally for anyone.

Reply:
@samumaretiya I did test this one which is how I noticed it wasn't working. If you notice here, https://github.com/Nofraud/nofraud_magento_checkout/blob/a9f71ea18692ed682a2e8446c94548b58467ed87/Helper/Data.php#L133 the code is expecting `prod`.  

I installed the changes and then simulated a clean config for this field by running `DELETE FROM core_config_data WHERE value = 'nofraud/advance/list_mode'`on the DB to clear any DB setting (since it takes precedence over config.xml). At that point, I put a log entry [here](https://github.com/Nofraud/nofraud_magento_checkout/blob/a9f71ea18692ed682a2e8446c94548b58467ed87/Helper/Data.php#L133). With the changes, the code returned Production which would not match and the scripts did not load as a result.

This is difficult to show on a screen recording, but we can meet later about it.